src/codeu/chat/client/commandline/Chat.java

```diff
@@ -437,6 +440,43 @@ public void invoke(List<String> args) {
 				System.out.format("  Id   : UUID:%s\n", user.user.id);
 			}
 		});
+		
+		// STATUS UPDATE
+		// 
+		// Allows user to see status update
+		// A status update includes : 
+		// 1) The number of messages that have been send in each of the user's 
+		//    conversation interests since the last updates 
+		// 2) A list of conversations created and conversations added to by each 
+		//    of the user's user interests 
+		// 
+		panel.register("status-update", new Panel.Command() { 
+			@Override
+			public void invoke(List<String> args) { 
+				HashMap<String, Integer> convoUpdates = user.CONVOstatusUpdate(); 
+				HashMap<Uuid, ArrayList<ArrayList<String>>> userUpdates = user.USERstatusUpdate();
+				int CONVOS_CREATED_ARRAY = 0; 
```

Comment:
Maybe make these constants a public static field in UserContext, that way you do not need to declare them more than once.